tests/webapp/api/test_performance_data_api.py

```diff
@@ -709,3 +711,40 @@ def test_filter_out_retriggers():
 
     filtered_data = PerformanceSummary._filter_out_retriggers(copy.deepcopy(no_retriggers_data))
     assert filtered_data == no_retriggers_data
+
+
+def test_alert_summary_tasks_get(client, test_perf_alert_summary, test_perf_data):
+    create_perf_alert(
+        summary=test_perf_alert_summary,
+        series_signature=test_perf_data.first().signature,
+        related_summary=test_perf_alert_summary,
+        status=PerformanceAlert.REASSIGNED,
+    )
+    resp = client.get(
+        reverse('performance-alertsummary-tasks') + '?id={}'.format(test_perf_alert_summary.id)
+    )
+    assert resp.status_code == 200
+    assert resp.json() == {
+        "id": test_perf_alert_summary.id,
+        "tasks": [
+            "Mochitest Browser Chrome",
+            "Inari Device Image Build",
+            "B2G Emulator Image Build",
+            "Nexus 4 Device Image Build",
+        ],
+    }
+
+
+def test_alert_summary_tasks_get_failure(client, test_perf_alert_summary):
+    # verify that we fail if PerformanceAlertSummary does not exist
+    not_exist_summary_id = -1
```

Comment:
The current variable is unnecessary,  but I used it because I wanted to emphasize that it's an id that doesn't exist.
Let me know if we don't use the variable. I'll write it right at the URL below without the variable.
example: `reverse('performance-alertsummary-tasks') + '?id=-1'`

Reply:
Here is an example about how to test the code for objects that don't exist: https://github.com/mozilla/treeherder/blob/5252b4763ca0178f2b50faf980f915c06ac3f8fb/tests/model/test_job_notes.py#L4

Reply:
Thanks :)
I'll write a better test code by referring to it.